enhancements/20220531-storage-tiering.md

```diff
@@ -0,0 +1,131 @@
+# Storage Tiering
+
+## Summary
+Expose storageClass to users. Users can define storage tiers and replicas in storageClass, and then select storageClass to specify storage tiers and replicas when creating a volume or image
+
+### Related Issues
+https://github.com/harvester/harvester/issues/2147
+
+## Motivation
+
+### Goals
+Allow users to define storage tiers based on:
+
++ Node tag selectors
++ Disk tag selectors
+
+### Non-goals [optional]
+
+## Proposal
+
+### User Stories
+
+Use a mix of storage hardware at different performance levels to
+
++ optimize cost
++ provide predictable performance
+
+### User Experience In Detail
+
+#### 1 Node Tag
+
+##### 1.1 Edit Node Tag
+`Hosts` > host > `Edit Config` > `Storage`
+
+![edit node tag](./20220531-storage-tiering/edit-node-tag.png)
+
+##### 1.2 Show Node Tag
+`Hosts` > host > `Storage`
+
+![show node tag](./20220531-storage-tiering/show-node-tag.png)
+
+#### 2 Disk Tag
+
+##### 2.1 Edit Disk Tag
+`Hosts` > host > `Edit Config` > `Storage`
```

Comment:
Do we need NDM's help to add some tags during the "add disk" stage?

Reply:
@bk201 @Vicente-Cheng 
It's a good idea, we can make it as a new NDM enhancement. but maybe not in v1.1.0? 
cc @guangbochen